doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -1881,6 +1881,59 @@ PSA Crypto support
          - Supported
          - Supported
 
+
+MCUboot bootloader
+==================
+
+.. toggle::
+
+   .. list-table::
+       :widths: auto
+       :header-rows: 1
+
+       * -
+         - nRF52810
+         - nRF52811
+         - nRF52820
+         - nRF52832
+         - nRF52833
+         - nRF52840
+         - nRF5340
+         - nRF54H20
+         - nRF54L15
+         - nRF9131
+         - nRF9151
+         - nRF9160
+         - nRF9161
+       * - **Immutable Bootloader as part of build**
```

Comment:
Doesn't the first row overlap with what's under the NSIB section (though the column for nRF54L15 isn't updated in the old content)?

Reply:
MCUboot might be instanced as immutable or updatable. Bootloader->MCUboot